tests/interactions_single_mesons/__main__.py

```diff
@@ -106,6 +106,14 @@
     'electron': theta,
 })
 
+# kind =
+# 0: I_coll = A + f_1 * B
+# 1: I_coll = A
+# 2: I_coll = f_1 * B
+# 3: I_coll = I_coll = A_vacuum_decay + f_1 * B_vacuum_decay
+# 4: I_coll = A_vacuum_decay
+# 5: I_coll = f_1 * B_vacuum_decay
+kind = 5
```

Comment:
I think currently `f_1` is not included into I_coll. I guess this might be conflicting with the decay tests. Does it?